locales/en.json

```diff
@@ -207,5 +207,30 @@
   "If this information doesn’t match your records, you can’t change it here.": "If this information doesn’t match your records, you can’t change it here.",
   "You should not use this service to file income information you believe to be incorrect.": "You should not use this service to file income information you believe to be incorrect.",
   "Language Selection": "Language Selection",
-  "Language selection: French": "Language selection: French"
-}
+  "Language selection: French": "Language selection: French",
+  "Review and file tax return": "Review and file tax return",
+  "Your annual benefits": "Your annual benefits",
+  "Detailed breakdown of benefits": "Detailed breakdown of benefits",
+  "Canada child benefit": "Canada child benefit",
+  "Climate action incentive": "Climate action incentive",
+  "GST/HSTC": "GST/HSTC",
+  "Trillium": "Trillium",
+  "Total Benefits": "Total Benefits",
+  "Your projected refund": "Your projected refund",
+  "Total Income (150)": "Total Income (150)",
+  "Childcare Expenses (214)": "Childcare Expenses (214)",
+  "Net Income (236)": "Net Income (236)",
+  "Taxable Income (260)": "Taxable Income (260)",
+  "Basic Personal Amount (300)": "Basic Personal Amount (300)",
+  "Age Amount (301)": "Age Amount (301)",
+  "Pension (314)": "Pension (314)",
+  "Disability (316)": "Disability (316)",
+  "Amounts transfered from spouse (326)": "Amounts transfered from spouse (326)",
+  "Medical Expenses (330/331)": "Medical Expenses (330/331)",
+  "Net Federal tax (420)": "Net Federal tax (420)",
+  "Net Provincial tax (428)": "Net Provincial tax (428)",
+  "Total Payable (435)": "Total Payable (435)",
+  "Total Deducted (437)": "Total Deducted (437)",
+  "Total Credits (482)": "Total Credits (482)",
+  "Total Refund (484)": "Total Refund (484)"
+}
```

Comment:
It might be easier to do a final commit where you delete these new keys, because otherwise it's gonna cause a bunch of conflicts with other PRs.

Might be better to get the code changes in first and then update the locales file in a separate PR.